<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dispertech - Fiber Alignment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>200</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="button_fiber_led">
         <property name="text">
          <string>LED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>250</height>
          </size>
         </property>
         <property name="title">
          <string>Mirror</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="button_focus_plus">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_focus_minus">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="arrows_widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <widget class="QPushButton" name="button_up">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>0</y>
               <width>31</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/Icons/arrow_top.png</normaloff>:/icons/Icons/arrow_top.png</iconset>
             </property>
            </widget>
            <widget class="QPushButton" name="button_down">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>60</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/Icons/arrow_bottom.png</normaloff>:/icons/Icons/arrow_bottom.png</iconset>
             </property>
            </widget>
            <widget class="QPushButton" name="button_right">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>27</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/Icons/arrow_right.png</normaloff>:/icons/Icons/arrow_right.png</iconset>
             </property>
            </widget>
            <widget class="QPushButton" name="button_left">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>30</y>
               <width>31</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/Icons/arrow_left.png</normaloff>:/icons/Icons/arrow_left.png</iconset>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Speed: </string>
            </property>
            <property name="buddy">
             <cstring>motor_speed_line</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="motor_speed_line"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Exposure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camera_exposure_line"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="camera_gain_line"/>
       </item>
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_core_button">
         <property name="text">
          <string>Save Core</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_laser_button">
         <property name="text">
          <string>Save Laser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fiber_core_position"/>
       </item>
       <item>
        <widget class="QLineEdit" name="laser_center_position"/>
       </item>
       <item>
        <widget class="QCheckBox" name="laser_track">
         <property name="text">
          <string>Laser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fiber_center_checkbox">
         <property name="text">
          <string>Fiber Center</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="camera_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Monitor the Fiber End</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_move_up">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move mirror up</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="action_move_down">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="action_move_left">
   <property name="text">
    <string>Move Left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="action_move_right">
   <property name="text">
    <string>Move Rigth</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
